edit-post/components/plugin-post-publish-panel/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Defines extensibility slot for post-publish sidebar.
+ *
+ * Defines slot to be used by plugins to insert content in the post-publish sidebar
+ * (which appears after a user fully publishes a post).
+ *
+ * @see /edit-post/README.md 
+ *
+ * @file   This files defines the PluginPostPublishPanel extension
+ * @author ChrisShultz
```

Comment:
We don't use `@author` in the project.

Reply:
Good to know. I'll kill it. 

Should I be consulting any standards other than what's here? - https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript

Reply:
Fixed